pkg/agent/openflow/network_policy.go

```diff
@@ -153,6 +153,30 @@ func (m *conjunctiveMatch) generateGlobalMapKey() string {
 	return fmt.Sprintf("table:%d,type:%d,value:%s", m.tableID, m.matchKey, m.matchValue)
 }
 
+type changeType int
```

Comment:
Originally I use changeType, because this type is used to mark the change type of the OpenFlow entry, the action and the conjunctiveMatchContext. So I didn't add any prefix for the changeType.
I think `flowChangeType` might not be suitable to describe changes in other objects.